README.rst

```diff
@@ -37,20 +44,44 @@ Installation
 1. Navigate to the ``backend`` directory.
 2. Initialize and install dependencies using Poetry.
 
+    cd backend
 
+    poetry install
+
+3. Navigate to the ``infra`` directory.
+4. Initialize and install dependencies using Poetry.
+
+    cd ../infra
```

Comment:
Is that step necessary given p.3 Navigate to infra directory?

Reply:
I added it to make it more aligned with steps 1 and 2

Reply:
Changed it to step by step representation